components/dashboard/src/Login.tsx

```diff
@@ -126,6 +136,12 @@ export function Login() {
                         </div>
                     </div>
                 </div>
+                {emailTaken && (<div className="bg-gitpod-kumquat-light rounded-md p-3 text-gitpod-red text-sm mb-2">
```

Comment:
**suggestion**: Can we make this alert component dismissable?  WDYT?

Reply:
Given that the Login page is just a stop between here and there, I'd personally never try to dismiss it, but if you think this is useful 🤷🏻‍♂️ let's add it. Also it's dismissed on next try to connect.

Reply:
It should be fine as non-dismissible for now as the task is blocked if no action is taken. Usually, an alert component is non-dismissible when the user is prevented from completing a task until a system-condition is met. Since this is user-driven and not relying on a system condition, a dismissible alert is probably the correct approach here. 💡 

However, we can tackle this in a follow up issue going through all alerts and make sure we make dismissible the ones needed. Your call! 📞

Reply:
> However, we can tackle this in a follow up issue going through all alerts and make sure we make dismissible the ones needed. Your call! 📞

sounds good!